app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -100,6 +100,11 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
         getHeaderViewModel().profile.set(it)
         getFooterViewModel().isAdmin.set(it.isAdmin)
         binding.administratorControlsLinearLayout.setOnClickListener {
+          if (getFooterViewModel().isAdministratorControlsSelected.get()!!) {
```

Comment:
Avoid using `!!`. Instead you can write it as 
`if (getFooterViewModel().isAdministratorControlsSelected.get() == true) {`

Reply:
@rt4914 Thanks for the suggestion, updated.